netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -465,8 +466,10 @@ private NettyClientTransport newTransport(ProtocolNegotiator negotiator, int max
     }
     NettyClientTransport transport = new NettyClientTransport(
         address, NioSocketChannel.class, new HashMap<ChannelOption<?>, Object>(), group, negotiator,
-        DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize, keepAliveTimeNano, 1L, false, authority,
-        userAgent);
+        DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize,
+        KeepAliveManager.clampKeepAliveTimeInNanos(keepAliveTimeNano),
```

Comment:
I don't follow. I only see values `1000` and INF used. If `1000` is too small, make it larger.